rust/lance/src/dataset.rs

```diff
@@ -487,13 +487,17 @@ impl Dataset {
             None,
         );
 
+        let manifest_config = ManifestWriteConfig {
+            use_stable_row_ids: params.enable_experimental_stable_row_ids,
```

Comment:
experimental & stable feels like an odd claim.

Reply:
Fair. What do you think of `enable_move_stable_row_ids`? We can label as experimental in the docs. And I like the `move` specifier to help connote they aren't stable in all situations yet.

Reply:
Renamed the feature as "Move-Stable Row Ids".